.github/workflows/test.yaml

```diff
@@ -784,6 +788,11 @@ jobs:
               export NO_DELTA=1
           fi
 
+          # Add openstack backend definition to spread.yaml
+          if [ "${{ matrix.backend }}" = openstack ]; then
+              ./tests/lib/spread/add-backend tests/lib/spread/backend.openstack.yaml spread.yaml
```

Comment:
is this just a workaround for folks having spread and not spread+ ?

Reply:
Yes, it's about retaining compatibility locally.